index.js

```diff
@@ -3,15 +3,23 @@ import {bigram} from 'n-gram'
 /**
  * Get the edit-distance according to Dice between two values.
  *
- * @param {string} value
- * @param {string} alternative
+ * @param {string|string[]} value
+ * @param {string|string[]} alternative
  * @returns {number}
  */
 export function diceCoefficient(value, alternative) {
   var value_ = String(value).toLowerCase()
   var alt = String(alternative).toLowerCase()
```

Comment:
I’m quite sure this is making your later code never run though?
It does two things: cast to string, and lowercase.
The casting could be done when the value isn’t a string
The lowercase could be done for each bigram maybe?

Reply:
Thanks for the quick review! Ah yeah, there is a bug, but for a different reason -- the later code is run because `left` and `right` are assigned to the original input params (`value` and `alternative`), rather than `value_` and `alt`. However, the bigram inputs wouldn't be case-insensitive. Made another commit that fixes the case sensitivity, but still probably not optimal from a readability perspective. Will try to come up with a more pleasant to read refactoring.

Reply:
Okay I think I've got a more readable solution committed.